src/askscript/__tests__/00-documentation-examples/documentation01-complete_example.ask

```diff
@@ -8,7 +8,7 @@ ask(arg1:int):string {
 	  n:multiply(factorial(n:minus(1)))
 	}
 	
-  const sum:int(int, int) = fun(a:int, b:int):int {
```

Comment:
Tests in 00-documentation-examples mirror exactly examples at the AskScript documentation.
If you remove the type here, please remove it also at https://www.notion.so/AskScript-Human-Friendly-Language-for-AskQL-7c5e382df87a4015b44985391861f6a2

Reply:
Let's wait with updating the documentation until the next release. I'm going to add a [CHANGELOG.md](https://changelog.md/) describing the next version and listing required changes to the documenation.